sdk/attestation/attestation/rollup.config.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import rollup from "rollup";
+import nodeResolve from "rollup-plugin-node-resolve";
+import sourcemaps from "rollup-plugin-sourcemaps";
+
+/**
+ * @type {rollup.RollupFileOptions}
+ */
+const config = {
+  input: "./esm/attestationClient.js",
+  external: ["@azure/core-http", "@azure/core-arm"],
```

Comment:
we still need to fix the codegen to remove core-arm it looks like

Reply:
Filed https://github.com/Azure/autorest.typescript/issues/807 . Here, I will remove it manually